compiler+runtime/src/cpp/jank/runtime/module/loader.cpp

```diff
@@ -298,6 +298,45 @@ namespace jank::runtime::module
     }
   }
 
+  static void
+  register_module_path(native_unordered_map<jtl::immutable_string, loader::entry> &entries,
+                       jtl::immutable_string const &paths,
+                       bool skip_jar = false)
```

Comment:
Please don't use default params. Once one is added, everything after it needs to be defaulted as well. Either use an overload which doesn't take the param and then calls this one or just require the param on all calls.

Also, please make `skip_jar` const.